RecoEcal/EgammaClusterAlgos/src/PreshowerPhiClusterAlgo.cc

```diff
@@ -90,7 +90,6 @@ reco::PreshowerCluster PreshowerPhiClusterAlgo::makeOneCluster(ESDetId strip,
   Point pos(x_pos,y_pos,z_pos);
 
   reco::PreshowerCluster cluster = reco::PreshowerCluster(Eclust, pos, usedHits, plane);
-  used_strips = used_s;
```

Comment:
I thought about that but I did not follow through with that because of consistency. In the same file, see https://github.com/uhussain/cmssw/blob/bbee4e7c154042691b8676d4ccb096958cde2385/RecoEcal/EgammaClusterAlgos/src/PreshowerPhiClusterAlgo.cc#L23
So for consistency, if I remove used_s everywhere its used in this file and the corresponding header file, I might need to do the same for this other variable which doesn't raise any errors. And that will mean a lot of deletions and insertions compared to just one deletion as I have now. @cvuosalo do you have any other comments about this?